src/agents/quest_agent.py

```diff
@@ -313,6 +316,9 @@ def evaluate_solution(
             required_roll = 0.1
         else:
             required_roll = 0.5
+        required_roll = 1 - (
+            (1 - required_roll) / server_settings.problem_solution_difficulty
```

Comment:
thought: i wonder if this will be "easy" enough for people to adjust, or if `problem_solution_difficulty` will mainly be left alone.

Reply:
I tried to make it as easy as I could think of (2 doubles the default difficulty, 0.5 halves it etc) but like you I remain dubious.